geth-patches/README.md

```diff
@@ -9,44 +9,36 @@ We try to minimize number and amount of changes in those patches as much as poss
 
 # Patches
 
- - `0000-accounts-hd-keys.patch` — adds support for HD extended keys (links/docs?)
- - `0002-les-api-status.patch` — adds StatusBackend into LES code (need to be inspected, some things can and should be done outside of les code)
- - `0003-dockerfiles-wnode-swarm.patch` — adds Dockerfiles (who uses this?)
- - `0004-whisper-notifications.patch` — adds Whisper notifications (need to be reviewed and documented)
- - `0006-latest-cht.patch` – updates CHT root hashes, should be updated regularly to keep sync fast, until proper Trusted Checkpoint sync is not implemented as part of LES/2 protocol.
- - `0007-README.patch` — update upstream README.md.
- - `0008-tx-pool-nonce.patch` - On GetTransactionCount request with PendingBlockNumber get the nonce from transaction pool
+ - [`0000-accounts-hd-keys.patch`](./0000-accounts-hd-keys.patch) — adds support for HD extended keys (links/docs?)
+ - [`0002-les-api-status.patch`](./0002-les-api-status.patch) — adds StatusBackend into LES code (need to be inspected, some things can and should be done outside of les code
+ - [`0003-dockerfiles-wnode-swarm.patch`](./0003-dockerfiles-wnode-swarm.patch) — adds Dockerfiles (who uses this?)
+ - [`0004-whisper-notifications.patch`](./0004-whisper-notifications.patch) — adds Whisper notifications (need to be reviewed and documented)
+ - [`0006-latest-cht.patch`](./0006-latest-cht.patch) – updates CHT root hashes, should be updated regularly to keep sync fast, until proper Trusted Checkpoint sync is not implemented as part of LES/2 protocol.
+ - [`0007-README.patch`](./0007-README.patch) — update upstream README.md.
+ - [`0008-tx-pool-nonce.patch`](./0008-tx-pool-nonce.patch) - On GetTransactionCount request with PendingBlockNumber get the nonce from transaction pool
+ - [`0010-geth-17-fix-npe-in-filter-system.patch`](./0010-geth-17-fix-npe-in-filter-system.patch) - Temp patch for 1.7.x to fix a NPE in the filter system
```

Comment:
Not yet sure if "17" is a good abbreviation for version 1.7.x. But that's more a topic about naming schemes to manage different versions.

Reply:
yeah, I agree.
hope this patch will go away soon anyway :)